Tools/autotest/arducopter.py

```diff
@@ -1564,6 +1564,110 @@ def fly_nav_takeoff_delay_abstime(self):
         if ex is not None:
             raise ex
 
+    def fly_zigzag_mode(self):
+        '''test zigzag flight mode'''
+        self.progress("Waiting for location")
+        start = self.mav.location()
+        self.mavproxy.send('switch 6\n')  # stabilize mode
+        self.mav.wait_heartbeat()
+        self.wait_mode('STABILIZE')
+        self.progress("Waiting reading for arm")
+        self.wait_ready_to_arm()
+
+        old_pos = self.mav.recv_match(type='GLOBAL_POSITION_INT', blocking=True)
+        print("old_pos=%s" % str(old_pos))
+
+        self.context_push();
+
+        ex = None
+        try:
+            # set channel 8 for ZigZag SaveWP
+            self.set_parameter("RC8_OPTION", 61)
+            self.mavproxy.send('mode loiter\n') # change to loiter mode
```

Comment:
self.mavproxy.send('switch 5\n')

Reply:
ok, changed to "self.mavproxy.send('switch 5\n')  # loiter mode"